packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -1,5 +1,9 @@
 export default {
 	async fetch(request, env) {
-		return new Response("Hello from Asset Server Worker 🚀");
+		const url = new URL(request.url);
+		let { pathname } = url;
+
+		const content = await env.__STATIC_CONTENT.get(pathname);
```

Comment:
If the asset does not exist what does `get()` return? `null`?
If so we would need to deal with that specially to return a 404 response IMO.

Reply:
This is supposed to be handled in WC-2434, unless we strongly object to that

Reply:
Yeah, _lots_ still to add here — totally fine if that's being handled in another PR.